packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1046,7 +1042,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 			});
 		}
 
-		// TODO Remove after SPO picks up the latest build.
```

Comment:
This comment about SPO applies to the part of the `if` statement skipping over ClientLeave messages.  That case can't ever be removed as long as this layer is doing parsing, since there could be some old files with old ops in them.  When when we remove this JSON.parse altogether (in AB#12052), we'll need to take that into consideration as well.  I'm hoping that we'll only parse messages we are actually going to process, so won't have to deal with this case.  There's some other factors too, so we'll see.